python/repositories.bzl

```diff
@@ -130,6 +130,16 @@ filegroup(
     ),
 )
 
+cc_library(
+    name = "py_headers",
```

Comment:
I'm torn if this should be called `py_headers` or `headers`. Anyone have opinions?

Reply:
Our internal patch calls this `python_headers` fwiw, just to add to the bikeshedding. I don't have any real preference though.

nit: one slight difference that would be nice is to expose them as a `filegroup` too, and have the `cc_library` dep on that.

Reply:
if you give me a name for each target I'm happy to make the update

Reply:
`filegroup` was `includes` and `cc_library` was `python_headers`